pkg/contexts/oci/repositories/ocireg/repository.go

```diff
@@ -141,6 +144,22 @@ func (r *RepositoryImpl) getResolver(comp string) (resolve.Resolver, error) {
 				return "", "", nil
 			},
 			DefaultScheme: r.info.Scheme,
+			DefaultTLS: &tls.Config{
+				MinVersion: tls.VersionTLS13,
+				RootCAs: func() *x509.CertPool {
+					rootCAs, _ := x509.SystemCertPool()
+					if rootCAs == nil {
+						rootCAs = x509.NewCertPool()
+					}
+					if creds != nil {
+						c := creds.GetProperty(identity.ATTR_CERTIFICATE_AUTHORITY)
+						if c != "" {
+							rootCAs.AppendCertsFromPEM([]byte(c))
+						}
+					}
+					return rootCAs
+				}(),
+			},
```

Comment:
May be, we should add rootCAs, only if the credentials provide a ca, because by default the system certs are implicitly used.

Reply:
done